### PR TITLE
Fix testBug69579

### PR DESCRIPTION
**Pull request by seneramz**
### Summary
Fixes test testBug69579()

### Description
- Deregisters the existing AWS driver once the NonRegisteringDriver completes its test. 

### Additional Reviewers
@seneramz 
@congoamz 
@sergiyv-bitquill
@hsuamz

### CODE REVIEW

File: src/test/java/com/mysql/cj/jdbc/NonRegisteringDriverTest.java [hunk 1]
```diff
@@ -58,6 +59,14 @@ public void setAwsProtocolOnlyToFalse() {
         software.aws.rds.jdbc.mysql.Driver.setAcceptAwsProtocolOnly(false);
     }
 
+    @AfterAll
+    public static void cleanup() throws SQLException {
+        Driver test = DriverManager.getDriver("jdbc:mysql://localhost");
```
Comment (sergiyv-bitquill) [inline src/test/java/com/mysql/cj/jdbc/NonRegisteringDriverTest.java:64]: "test" isn't good variable name. It should be something meaningful.

Comment (samathaamz) [reply src/test/java/com/mysql/cj/jdbc/NonRegisteringDriverTest.java:64]: Yeap you're right. Will change that.

File: src/test/java/com/mysql/cj/jdbc/NonRegisteringDriverTest.java [hunk 2]
```diff
@@ -58,6 +59,14 @@ public void setAwsProtocolOnlyToFalse() {
...
     }
 
+    @AfterAll
+    public static void cleanup() throws SQLException {
+        Driver registeredDriver = DriverManager.getDriver("jdbc:mysql://localhost");
+        if (registeredDriver instanceof software.aws.rds.jdbc.mysql.Driver)
+        {
+            DriverManager.deregisterDriver(registeredDriver);
+        }
+    }
```
Comment (congoamz) [inline src/test/java/com/mysql/cj/jdbc/NonRegisteringDriverTest.java:69]: nit: looks like there isn't a blank line between } and the next test

Comment (samathaamz) [reply src/test/java/com/mysql/cj/jdbc/NonRegisteringDriverTest.java:69]: Good catch, added the new line

**Review by sergiyvamz — APPROVED** [review]

**Pull request merged**